PurchasesCoreSwift/Networking/ETagManager.swift

```diff
@@ -4,7 +4,9 @@
 
 import Foundation
 
+// TODO:(post-migration): make internal
 @objc(RCETagManager) public class ETagManager: NSObject {
+
     internal static let eTagHeaderName = "X-RevenueCat-ETag"
```

Comment:
Totally minor. `internal` is the access control by default, so I think it could be removed, couldn't it?